Sources/APNS/Application+APNS.swift

```diff
@@ -60,6 +59,52 @@ extension Application.APNS: APNSwiftClient {
         self.application.eventLoopGroup.next()
     }
 
+    public func batchSend(
+        rawBytes payload: ByteBuffer,
+        pushType: APNSwiftConnection.PushType,
+        to deviceToken: String...,
```

Comment:
Any reason we use `to deviceToken: String...,` here and then `to deviceToken: [String],` below?

They are semantically the same yeah? Might as well use the same thing in both cases.

Reply:
Yes. It's just that in some situations it might be more convenient to enumerate, so I added it too

Reply:
I agree we should probably offer both. Swift doesn't like converting between the two